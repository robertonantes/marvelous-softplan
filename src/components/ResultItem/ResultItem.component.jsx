import React from "react";
import { Wrapper, Image, Name } from "./ResultItem.styles";

const ResultItem = ({ name, image }) => {
  return (
    <Wrapper>
      <Image src={image} />
      <Name>{name}</Name>
    </Wrapper>
  );
};

export default ResultItem;
