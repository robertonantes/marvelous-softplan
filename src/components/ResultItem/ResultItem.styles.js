import styled from "styled-components";

export const Wrapper = styled.div`
  display: flex;
  width: 100%;
  margin-bottom: 8px;
`;

export const Image = styled.img`
  width: 80px;
  border: 2px solid #fff;
`;

export const Name = styled.h4``;
