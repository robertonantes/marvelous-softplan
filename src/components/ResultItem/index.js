export { default } from "./ResultItem.component";
