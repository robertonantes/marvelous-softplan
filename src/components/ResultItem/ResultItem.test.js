import React from "react";
import { render } from "@testing-library/react";
import ResultItem from ".";

const data = {
  image: "image-path",
  name: "Result name",
};

describe("ResultItem Component", () => {
  it("Renders result info", () => {
    const { asFragment } = render(
      <ResultItem image={data.image} name={data.name} title="prop" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
