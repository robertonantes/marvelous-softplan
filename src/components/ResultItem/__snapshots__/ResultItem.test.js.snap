// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultItem Component Renders result info 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm KEoCo"
    title="prop"
  >
    <img
      class="sc-AxirZ iBklgE"
      src="image-path"
    />
    <h4
      class="sc-AxiKw iVJYNO"
    >
      Result name
    </h4>
  </div>
</DocumentFragment>
`;
